### ISSUE TITLE
Some questions about training and dataset.

### ISSUE THREAD

**Issue by jackyyang9** [korrawe/halo#6]
Hello, Mr.Karunratanakul,

HALO is a nice work, especially the skeleton canonicalization part in paper.

It offers another feasible way to parameterize the pose.

In fact, I am reproducing HALO in FreiHAND dataset and I have some naive questions.

1): FreiHAND dataset doesn't has evaluation dataset, so I cannot observe the val loss in training.

  My question is how can I know the training should be stopped from the real-time trainning loss?

2): Could you offer the Supplemently material you mentioned in paper? 

  I wish to know more details about the experiment.

3): I obtained YT3D dataset from official website.

  However, they didn't provide joints and mano parameters.

  And I notice that in line 32 of prepare_data_from_mano_param_keypoints.py , there is "default='../../data/youtuhand_raw'".

  What is that? 
  How can I get this?

Sincerely looking forward to your reply.

Best wishes.

**Comment by korrawe** [comment]
Hi,

1. You can split a portion of the training data for validation.
2. The supplementary is at the end of the paper on arxiv
3. You can get key points using the J_regressor matrix in the MANO model. If you want MANO parameters, you can fit a MANO hand to the provided surface points using the fitting script in prepare_data_from_mano_param_keypoints.py. Other than that you can also use a method like [SMPLify](https://github.com/vchoutas/smplify-x#fitting).

Best,

**Comment by jackyyang9** [comment]
Thanks for your quick reply !
I already found the supplemental material in arxiv version and thansk for other answers. 
Best wishes to all your paper subbmision!

**Comment by jackyyang9** [comment]
Thanks again.
However I still have questions:
1. I noticed that in prepare_data_from_mano_param_keypoints.py, you regarded the bones' translation matrixs of palm. If I want to get full transformation matrixs for each bone. How can I get it.
2. If we have the transformation matrixs, why we still need the root_rotation_matrix?
